src/main/view/StyleManager.cpp

```diff
@@ -7,9 +7,42 @@
 
 #include "StyleManager.h"
 
+#include <algorithm>
+
+#if defined(_WIN64) || defined(_WIN32)
+//Windows specific code
+#include <windows.h>
+//End Windows specific code	  
+#endif
+
 using namespace gs;
 
-const std::string StyleManager::assetPath = "assets/";
+const std::string StyleManager::assetPath = "/assets/";
+
+namespace {
+
+std::string getExecutablePath() {
+	static std::string execPath;
+
+	if (execPath.size() > 0) {
+		return execPath;
+	}
+
+	#if defined(_WIN64) || defined(_WIN32)
+		HMODULE hModule = GetModuleHandleW(NULL);
+		WCHAR path[MAX_PATH];
+		int lengthWcharPath = GetModuleFileNameW(hModule, path, MAX_PATH);
+		int lengthStringPath = WideCharToMultiByte(CP_UTF8, 0, path, lengthWcharPath, NULL, 0, NULL, NULL);
+		execPath = std::string(lengthStringPath, 0);
+		WideCharToMultiByte(CP_UTF8, 0, path, lengthWcharPath, &execPath[0], lengthStringPath, NULL, NULL);
+		std::replace(execPath.begin(), execPath.end(), '\\', '/');
+		execPath = execPath.substr(0, execPath.find_last_of("/"));
```

Comment:
Does this line just remove the last forward slash and everything after it? If so, could we retain the final forward slash and just lose everything after it. Then we won't need the leading forward slash on the const above.